final/scripts/README.md

```diff
@@ -0,0 +1,32 @@
+## Scripts to Perform Final Analysis
+
+This repository stores the script files to for all final analysis. 
+
+Scripts for main final data analysis (listed in the order that they should be run): 
+
+- `smooth_final.py`: Spatially smooths voxels. 
+- `convolution_final.py`: Creates convolved and time-shifted HRFs for each subject's voxel time courses. 
+- `glm_final.py`: Runs linear regression models with linear drift, Fourier, and PCA terms. 
+- `normality_final.py`: Tests the assumption of normality by running a Shaprio-Wilks test on each voxel's residuals for each subject. 
+- `bh_t_beta_final.py`: Analysis of Benjamini-Hochberg corrected "p-values" and t-statistics and beta coefficient estimates above and below certain thresholds. 
+
+
+Scripts for supplementary final exploratory data analysis: 
+
+- `selection_final.py`: Selects predictors for linear models based on AIC, BIC, and adjusted R-squared. 
+- `parameter_selection_final.py`: Performs parameter selection for Benjamini-Hochberg, beta coefficient estimate, t-statistic and "p-value" clustering for a single subject of choice. 
+
+Scripts to reproduce additional figures for the report and slides that are not generated by the main and supplementary final data analysis scripts: 
+
+- `pca_final.py`: Generates PCA-related plots for the paper. 
+
+Where do these go? 
+
+- `ben_final.py`: Something to do with parameter tuning? 
+- `conclusion_final.py`: Is this file overwritten by `glm_final.py`?
+- `hypothesis_final.py`: Is this file overwritten by `bh_t_beta_final.py`?
+- `image_overlay_final.py`: Something to do with parameter tuning? 
+
```

Comment:
I think we can remove conclusion_final and hypothesis_final (but we should check with Hiro. Image_overlay needs so thoughts from people but it probably will produce the final images. I think ben_final produces so images of the multiple comparison methods per individual-> I think I made it because I hadn't doing the same plots over and over again. Probably remove/ change in the end